src/components/Footer/Footer.svelte

```diff
@@ -24,6 +25,11 @@
 		<a href="mailto:bountymanager@galaniprojects.com">
 			<img width="40" height="40" src={LogoEmail} alt="Email Logo" />
 		</a>
+		<a
+			href="https://github.com/galaniprojects/Polkadot-Bounty-Manager.git"
```

Comment:
```suggestion
			href="https://github.com/galaniprojects/Polkadot-Bounty-Manager"
```
GitHub redirects me to a different URL